atom/browser/net/atom_network_delegate.cc

```diff
@@ -484,8 +484,8 @@ int AtomNetworkDelegate::HandleResponseEvent(
   callbacks_[request->identifier()] = std::move(callback);
 
   ResponseCallback response =
-      base::Bind(&AtomNetworkDelegate::OnListenerResultInUI<Out>,
-                 base::Unretained(this), request->identifier(), out);
+      base::BindRepeating(&AtomNetworkDelegate::OnListenerResultInUI<Out>,
```

Comment:
can this be `Once`?

Reply:
I wouldn't say so since the ReponseListener code is locked into repeatability; i can convert it to once and then adapt for repeating but at that point it may not be worth it 🤔

Reply:
ah ok, i'm just sort of trying to infer onceness from context

Reply:
the `ResponseListener` is repeatable but the `ResponseCallback` can be once, @nornagon is referring to the second.